hieraserver/rest.go

```diff
@@ -68,9 +68,13 @@ func startServer(cmd *cobra.Command, _ []string) {
 	configOptions := map[string]px.Value{
 		provider.LookupProvidersKey: types.WrapRuntime([]hieraapi.LookupKey{provider.ConfigLookupKey, provider.Environment})}
 
-	if config != `` {
-		configOptions[hieraapi.HieraConfig] = types.WrapString(config)
+	var cfgfile string
+	if config == `` {
+		cfgfile = "/hiera/hiera.yaml"
```

Comment:
Simpler to just pass the default as a third argument on line 50.

Reply:
Ah that's much better!